src/telemetry/telemetryEvents.ts

```diff
@@ -627,6 +634,41 @@ export class PresetConnectionEditedTelemetryEvent
   }
 }
 
+/** Reported when the extension side panel is opened. VSCode doesn't expose
+ * a subscribable event for this, so we're inferring it by subscribing to
+ * treeView.onDidChangeVisibility for all the extension treeviews and throttling
+ * the events.
+ */
+export class PanelOpenedTelemetryEvent implements TelemetryEventBase {
```

Comment:
```suggestion
export class SidePanelOpenedTelemetryEvent implements TelemetryEventBase {
```
could be better to have a more strict relationship between naming of the type and the class itself